components/rmrk/Gallery/Gallery.vue

```diff
@@ -203,23 +192,21 @@ export default class Gallery extends mixins(PrefixMixin, InfiniteScrollMixin) {
       return false
     }
     this.isFetchingData = true
-    const isRemark = this.urlPrefix === 'rmrk'
-    const query = isRemark
-      ? await import('@/queries/nftListWithSearch.graphql')
-      : await import('@/queries/unique/nftListWithSearch.graphql')
+    const query = await resolveQueryPath(this.urlPrefix, 'nftListWithSearch')
     const result = await this.$apollo.query({
       query: query.default,
       client: this.urlPrefix,
       variables: {
-        denyList: isRemark ? denyList : statemineDenyList,
-        orderBy: this.searchQuery.sortBy,
+        denyList: getDenyList(this.urlPrefix),
+        // orderBy: this.searchQuery.sortBy,
         search: this.buildSearchParam(),
```

Comment:
maybe use `get buildSearchParam`

Reply:
? :D it's used

Reply:
Oh, I mean convert it to computed field. For now, it is a method.